base_layer/core/src/transactions/transaction/transaction_output.rs

```diff
@@ -70,11 +71,29 @@ use crate::{
     },
 };
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[repr(u8)]
+pub enum TransactionOutputVersion {
+    DIBBLER = 0,
+}
+
```

Comment:
I think we need a `TransactionOutputVersion::GetCurrentVersion()` implementation here.